neonvm/controllers/virtualmachine_controller.go

```diff
@@ -74,6 +74,8 @@ type VirtualMachineReconciler struct {
 	client.Client
 	Scheme   *runtime.Scheme
 	Recorder record.EventRecorder
+
+	Metrics ReconcilerMetrics `exhaustruct:"optional"`
```

Comment:
TIL there's exhaustruct struct tags - nice! :D

Reply:
Is there a reason why it's needed here and not for `VirtualMachineMigrationReconciler` ?

Reply:
It's because of tests. There is a test that we have initialized a `VirtualMachineReconciler`.